packages/components/alert-dialog/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@spark-ui/alert-dialog",
+  "version": "1.0.0",
+  "description": "A modal dialog that interrupts the user with important content and expects a response.",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "vite build"
+  },
+  "dependencies": {
+    "class-variance-authority": "0.4.0",
```

Comment:
```suggestion
    "class-variance-authority": "0.5.2",
```

Reply:
We should also update CVA in dialog and tabs then :)

Reply:
Done